javascripts/app.js

```diff
@@ -18,6 +18,21 @@ var spell = new Gauntlet.SpellBook.Sphere();
 console.log("spell: ", spell.toString());
 
 
+$('#weapon-select').click(function(e) {     
+  if(e.target.parentNode.classList.contains('bareHands') || e.target.closest(".barehands")){       
+    PlayerOne.setWeapon(new Weapon());       
```

Comment:
Consider putting weapons into their own namespace on the `Gauntlet` object.

e.g. `Gauntlet.WeaponRack`